### ISSUE TITLE
MODX3: Error during installation through the console

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#14707]
### Summary
Errors appear during the installation process, at the stage **php7.1 transport.core.php**, although a few days ago this was not observed.

```
[2019-09-01 21:48:58] (ERROR @ /home/a/aleeviz8/m3/public_html/core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 792) Could not load class: modSystemSetting from mysql.modsystemsetting
Fatal error: Uncaught Error: Call to a member function fromArray() on null in /home/a/aleeviz8/m3/public_html/_build/data/transport.core.system_settings.php:2153
Stack trace:
#0 /home/a/aleeviz8/m3/public_html/_build/transport.core.php(242): include()
#1 {main}
  thrown in /home/a/aleeviz8/m3/public_html/_build/data/transport.core.system_settings.php on line 2153
```

**How I installed MODX3**
I used the hosting web-console:

cd m3/public_html/
rm -rf *
git clone https://github.com/modxcms/revolution .
git checkout 3.x
composer-php7.1 install
cd _build
cp build.config.sample.php build.config.php
cp build.properties.sample.php build.properties.php
php7.1 transport.core.php

### Environment
MODX 3
Server information:
Ubuntu
Apache: 2.4.10
MySQL: 5.7.21-20-beget
Nginx: 1.13.4
Perl: 5.26.1
PHP: 5.2 5.3 5.6 7.0 7.1 7.2 7.3
Python: 2.7

**Comment by Mark-H** [comment]
My bad, screwed something up with a merge earlier. See #14709.

**Comment by Ruslan-Aleev** [comment]
@Mark-H I tested the specified PR, the installation passes, but the manager panel as a result looks like this:

![manager](https://user-images.githubusercontent.com/12523676/64081696-7ad07580-cd15-11e9-840b-f8e8d157bb1e.png)

**Comment by Ruslan-Aleev** [comment]
Moreover, only the home section, through a direct link (for example, /manager/?a=system/settings) everything works, see gif:

![home](https://user-images.githubusercontent.com/12523676/64081726-021de900-cd16-11e9-89e3-90aee283cd63.gif)

**Comment by Mark-H** [comment]
Hmmm that's weird, I can't reproduce that.

**Comment by Mark-H** [comment]
Okay got it reproduced on an empty install now.

**Comment by Mark-H** [comment]
Another PR: #14710

**Comment by Ibochkarev** [comment]
Fixed in #14709 
@modxbot close
